### DISCUSSION TITLE
Get Amount of Free Memory on HP Prime?

### DISCUSSION THREAD

**Discussion by toml12953** [micropython/micropython#11517] [Using MicroPython]
I'd like to be able to get the amount of free memory in HP Prime's Python. The Prime has no gc library so I can't use that. Is there a way to do it?

**Comment by robert-hh** [comment]
You can run micropython.mem:inf(). Here on my HP Prime is tells me that about 1MB RAM are free.
Edit: But I on my HP I have the gc module.
